test/powershell/Modules/Microsoft.PowerShell.Security/certificateCommon.psm1

```diff
@@ -68,6 +68,19 @@ OksttXT1kXf+aez9EzDlsgQU4ck78h0WTy01zHLwSKNWK4wFFQM=
     return $certLocation
 }
 
+Function New-CertificatePassword
+{
+    # Hackery to create a new 10 digit random hex string as a password
+    $rand = [Random]::new()
+    $script:protectedCertPassword = ConvertTo-SecureString -Force -AsPlainText (((1..10).ForEach{ '{0:x}' -f $rand.Next(0xf) }) -join '')
+    return $script:protectedCertPassword
+}
+
+Function Get-CertificatePassword
+{
+    return $script:protectedCertPassword
```

Comment:
Should throw if $script:protectedCertPassword is not set or create a new one and return.